build.gradle

```diff
@@ -41,7 +45,7 @@ dependencies {
     annotationProcessor 'org.projectlombok:lombok'
     testImplementation 'org.springframework.boot:spring-boot-starter-test'
     testRuntimeOnly 'org.junit.platform:junit-platform-launcher'
-    implementation 'org.mifos:ph-ee-connector-common:1.9.1-SNAPSHOT'
+    implementation 'org.mifos:ph-ee-connector-common:0.0.0'
```

Comment:
Also, update this to the released version after the jfrog issue is fixed.

Reply:
Okay

Reply:
Using 0.0.0 as this [PR](https://github.com/fynarfin/ph-ee-connector-common/pull/3) has not been merged and new version is not released.